### ISSUE TITLE
[BUG] Release 0.6.4 giving compile error: "gdk_since_3_22.go:140:180: undefined: callback"

### ISSUE THREAD

**Issue by lgirma** [gotk3/gotk3#930]
**Describe the bug**
Running a the simple example provided in the home page used to work. But now getting compile errors.

**To Reproduce**
Compile the example from the "Sample Use" section in the README.md.

**Expected behavior**
Should compile OK.

**Error messages**
Getting: "`github.com/gotk3/gotk3@v0.6.4/gdk/gdk_since_3_22.go:140:180: undefined: callback`"

**Environment:**
Go version: 1.22.0 linux/amd64
GTK version: 3.24.38
Gotk version: 0.6.4
OS: Debian 12 on WSL
 
**Additional context**
The PR [#921](https://github.com/gotk3/gotk3/pull/921/files) may have broken it.

**Comment by anoraktrend** [comment]
confirmed on alpine edge

**Comment by RazerFord** [comment]
confirmed on Mint 21.2

**Comment by mintsoft** [comment]
Yeah that is definitely the cause, I missed the import when merging the code between the different PRs. This fixes it: https://github.com/gotk3/gotk3/pull/931

**Comment by kulothunganug** [comment]
experiencing the same issue
